drivers/volume/portworx/portworx.go

```diff
@@ -624,6 +624,12 @@ func (d *portworx) WaitDriverUpOnNode(n node.Node) error {
 		return err
 	}
 
+	//Check if PX pod is up
+	if d.schedOps.IsPXPodRunningOnNode(n) != true {
```

Comment:
nit: use ! instead of comparing with true

Reply:
Fixed.